windows-installer.bat

```diff
@@ -0,0 +1,128 @@
+@echo off
+echo Setup discord-twitter-video-embed by adryd325
+echo Script made by bagusnl with GitHub Copilot :P
+echo.
+echo Make sure chocolatey is installed
+echo and script running as administrator
+echo.
+
+:: check batch running as administrator
+:: https://stackoverflow.com/questions/4051883/batch-script-how-to-check-for-admin-rights
+:check_Permissions
+    echo Administrative permissions required. Detecting permissions...
+    
+    net session >nul 2>&1
+    if %errorLevel% == 0 (
+        echo Success: Administrative permissions confirmed.
+        goto chococheck
+    ) else (
+        echo Failure: Current permissions inadequate.
+    )
+    goto :end
+
+
+
+:chococheck
+:: check chocolatey installed
+:: https://stackoverflow.com/questions/5696907/how-to-check-if-a-program-is-installed-on-windows
+echo.
+echo Checking if chocolatey is installed...
+choco >nul 2>&1
+if %errorLevel% == 1 (
+    echo Success: Chocolatey is installed.
+    goto makedir
+) else (
+    echo Failure: Chocolatey is not installed.
+    echo.
+    echo Install chocolatey using this guide:
+    echo https://chocolatey.org/install
+    echo.
+    goto end
+)
+
+:makedir
+:: make installation directory
+mkdir C:\git
+cd /D C:\git
+
+:: install dependencies
+:: dependencies: git, ffmpeg, yt-dlp, nodejs
+echo.
+echo Installing dependencies...
+choco install git ffmpeg yt-dlp nodejs -y
+if %errorlevel% == 1 (
+    echo Failure: Dependencies not installed.
+    echo.
+    echo Install dependencies using this guide:
+    echo https://chocolatey.org/install
+    echo.
+    goto end
+) else (
+    echo Success: Dependencies installed.
+    goto gitclone
+)
+
+:: clone repo
+:gitclone
+:: delete git repo and clone again
+echo.
+echo Cleaning up previous installation...
+del /Q /F C:\Users\%username%\Desktop\dtve-runner.bat
+del /Q /F C:\Users\%username%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\windows-runner.bat
+rmdir /s /q C:\git\discord-twitter-video-embeds
+echo.
+echo Cloning repo...
+git clone https://github.com/bagusnl/discord-twitter-video-embeds
```

Comment:
This should be changed to refer to your repo, as it was referring to my repo because it needed the windows-runner.bat which wasn't in the main repo before
I'm really sorry